cypress/integration/jsEmployeesTableDOM.spec.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+Cypress.Commands.add('fillNewEmployeeForm',
+  (fullName, position, city, age, salary) => {
+    cy.get('[data-qa="name"]').type(fullName);
+    cy.get('[data-qa="position"]').type(position);
+    cy.get('[data-qa="city"]').select(city);
+    cy.get('[data-qa="age"]').type(age);
+    cy.get('[data-qa="salary"]').type(salary);
+    cy.get('button').contains('Save to table').click();
+  });
+
+Cypress.Commands.add('checkDataDoesntExist',
+  (fullName, salary) => {
+    cy.get('tbody').contains(fullName).should('not.exist');
+    cy.get('tbody').contains(salary).should('not.exist');
+  });
+
+Cypress.Commands.add('hasData', (inputName, compareValue) => {
+  cy.contains('tr', inputName).should(($column) => {
+    const array = [...$column].map((td) => td.innerText.split('\t'));
+
+    for (let i = 0; i < 5; i++) {
+      expect(array[0][i]).to.equal(compareValue[i]);
+    }
+  });
+});
+
+Cypress.Commands.add('isSortedASC', (columnName, columnNumber) => {
+  cy.contains(columnName).click();
+
+  cy.get(`tr:nth-child(n) td:nth-child(${columnNumber})`).then(($column) => {
```

Comment:
Create object like 
`const tableColumnsNames = {
    name: 1,
    position: 2,
    office: 3,
    age: 4,
    salary: 5
}` and use that object like `td:nth-child(${tableColumnsNames[columnNumber]})`